implementations/rust/ockam/ockam_entity/src/entity.rs

```diff
@@ -71,6 +61,10 @@ fn err<T>() -> Result<T> {
 impl Entity {
     pub fn create_profile(&mut self, vault_address: &Address) -> Result<Profile> {
         if let Res::CreateProfile(id) = self.call(CreateProfile(vault_address.clone()))? {
+            // Set current_profile_id, if it's first profile
```

Comment:
@jared-s what do you think?

Reply:
I think it should be ok, it will require explicit calls to create_profile before an entity can use a profile right? so basically we are taking away the default profile and making it lazily created. which i think is fine